packages/mocks/src/handlers.ts

```diff
@@ -11,7 +11,7 @@ const choose = (options: unknown[]) => {
 };
 
 const getItems = () =>
-  Array.from({ length: Math.random() * 10 }).map(() => ({
+  Array.from({ length: Math.random() * 10 + 1 }).map(() => ({
```

Comment:
that was annoying, huh? wanted to have the option when demoing, to show what no items would look like. not needed now I think